### PR TITLE
fix: sync task still running after DataNode has stopped

### PR DESCRIPTION
**Pull request by jaime0815**
issue: #38319
pr: #38377

### CODE REVIEW

File: pkg/util/paramtable/component_param.go [hunk 1]
```diff
@@ -946,6 +948,15 @@ This helps Milvus-CDC synchronize incremental data`,
 		Export:       true,
 	}
 	p.LocalRPCEnabled.Init(base.mgr)
+
+	p.SyncTaskPoolReleaseTimeoutSeconds = ParamItem{
+		Key:          "common.sync.taskPoolReleaseTimeoutSeconds",
+		DefaultValue: "60",
+		Version:      "2.4.19",
```
Comment (congqixia) [inline pkg/util/paramtable/component_param.go:955]: NIT: 2.4.18?

Comment (jaime0815) [reply pkg/util/paramtable/component_param.go]: fixed

**Review by congqixia — APPROVED** [review]
/lgtm

**Pull request merged**